templates/home.html

```diff
@@ -1,17 +1,18 @@
 {{ define "content" }}
 
-<h1 class="text-primary">HELLO GOLANG</h1>
-{{ include "partials/flashes" }}
 
-{{ if .data.authenticatedUser }}
-  <h2 class="greeting">Welcome {{ .data.email }}</h2>
-  <h2><a href="/keyword">Keyword</a></h2>
-  <form class="mt-8 space-y-6" action="/logout" method="POST">
-    <button type="submit">Logout</button>
-  </form>
-{{ else }}
-  <h2><a href="/login">Login</a></h2>
-  <h2><a href="/register">Register</a></h2>
-{{ end }}
+  <div class="home">
```

Comment:
This class name should be added to the `body` tag. So each controller can correspond to one screen:

```
<body class="home">
<body class="keyword">
<body class="register">
```

This cannot be a component for sure. But also you often forget [a basic rule of components](https://nimblehq.co/compass/development/code-conventions/css/):

> Class names must be composed of at least two words to make each element class name specific and more descriptive.

Reply:
Fixed in a90e792bc33581d3d5c17bb0ad0bc1fd65f04217

Reply:
Fixed in 7a25f26094e351e41c3bf9355ca1d1fef2e16ee8